inst/rmarkdown/templates/bioinformatics_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title_full: Template for preparing a submission to Bioinformatics using RMarkdown
+title_short: Bioinformatics Rmd Template
+
+# one of `Original Paper`, `Discovery Note`, `Application Note`, `Review`, `Letter to the Editor`, `Conference Paper`    
+subject_section: Application Note
+
+access_date:
+  - day: DD
+    month: MM
+    year: YY
+      
+author:
+  - name: Alice Anonymous
+    affiliation: "1,2"
+  - name: Bob Security
+    affiliation: "1"
+      
+address:
+  - code: 1
+    address: Some Institute of Technology, Department, Street, City, State, Zip
+  - code: 2
+    address: Another University Department, Street, City, State, Zip
+ 
+# Full email address to be given, preferably an institution email address.  
+corresponding_author_text: "To whom correspondence should be addressed. E-mail: bob\\@email.com"
+corresponding_author_email: "bob\\@email.com"
+  
+motivation: |
+  This section should specifically state the scientific question within the context of the field of study.
+  
+results: |
+  This section should summarize the scientific advance or novel results of the study, and its impact on computational biology.
+
+availability: |
+  This section should state software availability if the paper focuses mainly on software development or on the implementation of an algorithm. Examples are: 'Freely available on the web at XXX.' Website implemented in Perl, MySQL and Apache, with all major browsers supported'; or 'Source code and binaries freely available for download at URL, implemented in C++ and supported on linux and MS Windows'. The complete address (URL) should be given. If the manuscript describes new software tools or the implementation of novel algorithms the software must be freely available to non-commercial users. Authors must also ensure that the software is available for a full TWO YEARS following publication. The editors of Bioinformatics encourage authors to make their source code available and, if possible, to provide access through an open source license.
+
+supplementary_info: |
+  Supplementary data are available at Bioinformatics Online.
+
+output:
+  rticles::bioinformatics_article:
+    citation_package: natbib
+    
+bibliography: bibliography.bib
+# biblio-style: natbib
```

Comment:
On line `#65` in `template.tex`, the bibliography style has already been hard-coded, so I don't think this `biblio-style` field is necessary here. It can confuse users because it may give them a wrong impression that the `biblio-style` is customizable.

Reply:
@yihui Thanks, I agree with you. Could you directly change it in this PR web editor?

Reply:
Yes, I will.